contributors/devel/sig-node/cherry-picks.md

```diff
@@ -0,0 +1,115 @@
+# SIG Node Cherry-Pick Management Process
+
+Kubernetes supports multiple release versions simultaneously. How do new fixes
+make it into earlier versions, when we only develop one version at a time?
+
+We [cherry-pick] them!
+
+[cherry-pick]: https://git-scm.com/docs/git-cherry-pick
```

Comment:
not the best link. May be misleading. Maybe https://github.com/kubernetes/community/blob/master/contributors/devel/sig-release/cherry-picks.md?

Reply:
I already link that. None of the k8s docs explain what a "cherry-pick" is so I think it's valuable to include for context.

Reply:
same concept, but not the same implementation. I just don't want to confuse anybody.